test/integration/test_ibis.py

```diff
@@ -210,7 +202,7 @@ def test_nested_query(ibis_sqllite_db):
         },
     )
 
-    t = IbisTable(identifier='my_table', schema=schema)
+    t = Table(identifier='my_table', schema=schema)
```

Comment:
Renamed this to `Table` to bring in line with the way we use MongoDB `Collection`.